src/base/solvers/getCobraSolverParams.m

```diff
@@ -59,6 +59,9 @@
 valDef.saveInput = [];
 valDef.PbName = [solverType 'problem'];
 
+% GUROBI parameters
+valDef.method = -1; % solver method: -1 = automatic, 0 = primal simplex, 1 = dual simplex, 2 = barrier, 3 = concurrent, 4 = deterministic concurrent
+

```

Comment:
`method` looks to me like some general method switch (which other solvers have as well). We should either make this into a general setting or not create a cobra parameter here.

Reply:
This is only for Gurobi for now.